policyengine_us/variables/gov/states/mt/tax/income/exemptions/mt_exemptions.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class mt_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Number of Montana exemptions"
```

Comment:
```suggestion
    label = "Montana exemptions"
```
Or maybe value of but I think this is fine